debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -809,6 +809,17 @@ public static GtidNewChannelPosition parse(String value, String defaultValue) {
                     + DatabaseHistory.CONFIGURATION_FIELD_PREFIX_STRING + "' string.")
             .withDefault(KafkaDatabaseHistory.class.getName());
 
+    public static final Field USE_CURSOR_FETCH = Field.create("useCursorFetch")
```

Comment:
Do we really need that option? Can't we just rely on the pass-through `database.useCursorFetch` instead?